src/pages/ReportAvatar.tsx

```diff
@@ -23,21 +23,38 @@ type ReportAvatarProps = ReportAvatarOnyxProps & StackScreenProps<AuthScreensPar
 function ReportAvatar({report = {} as Report, route, policies, isLoadingApp = true}: ReportAvatarProps) {
```

Comment:
While at it, can you please migrate the `withOnyx` props to use the hook instead `useOnyx`